sky/backends/wheel_utils.py

```diff
@@ -129,7 +129,9 @@ def _build_sky_wheel() -> pathlib.Path:
 
         wheel_dir = WHEEL_DIR / hash_of_latest_wheel
         wheel_dir.mkdir(parents=True, exist_ok=True)
-        shutil.move(str(wheel_path), wheel_dir)
+        shutil.copy2(str(wheel_path), wheel_dir)
+        # if not os.path.exists(os.path.join(wheel_dir, os.path.basename(wheel_path))):
+        #     shutil.move(str(wheel_path), wheel_dir)
```

Comment:
Is this change mainly caused by sharing the `~/.sky` folder across multiple functions?

Reply:
I believe it's because we're moving the wheel from a `9p` to `tmpfs` type filesystem (or vice versa).